pkg/apis/core/v1/conversion.go

```diff
@@ -540,3 +541,38 @@ func Convert_core_PersistentVolumeSpec_To_v1_PersistentVolumeSpec(in *core.Persi
 func Convert_v1_PersistentVolumeSpec_To_core_PersistentVolumeSpec(in *v1.PersistentVolumeSpec, out *core.PersistentVolumeSpec, s conversion.Scope) error {
 	return autoConvert_v1_PersistentVolumeSpec_To_core_PersistentVolumeSpec(in, out, s)
 }
+
+func Convert_v1_LoadBalancerStatus_To_networking_LoadBalancerStatus(in *v1.LoadBalancerStatus, out *networking.LoadBalancerStatus, s conversion.Scope) error {
```

Comment:
Why are we open-coding conversion - this seems like it should be autogeneratable?